packages/frontend/src/components/Dialog/styles.module.scss

```diff
@@ -154,6 +154,7 @@ $paddingVertical: 20px;
 
 .footerActionButton {
   min-width: 120px;
+  white-space: nowrap;
```

Comment:
Another thing: the dialog does not look good on some languages

![image](https://github.com/user-attachments/assets/91219e5b-9c4e-497d-85a8-86461941a46b)

Reply:
can't we have the buttons have a minimal width and otherwise fit the text? and if the buttons do not fit into one line, let them flow in multiple lines?

Reply:
With a higher minimal-width all buttons get that width and then even the buttons with english text don't fit anymore. We would need a larger dialog width then depending on the text length

So as discussed in the issue this might be easier to solve in a column oriented button layout.
While shorter texts still fit in the buttons with whitespace: nowrap, longer texts don't.


And the problem mentioned in the comment above for small screens would also be solved.

Reply:
Same, this should be resolved by https://github.com/deltachat/deltachat-desktop/pull/4806.